internal/server/worker.go

```diff
@@ -190,19 +189,19 @@ func (w *Worker) clone() *Worker {
 
 // ActiveConnectionCount is the current number of sessions this worker is handling
 // according to the controllers.
-func (w *Worker) ActiveConnectionCount() uint32 {
-	return w.activeConnectionCount
+func (w *Worker) GetActiveConnectionCount() uint32 {
+	return w.ActiveConnectionCount
 }
 
 // CanonicalTags is the deduplicated set of tags contained on both the resource
 // set over the API as well as the tags reported by the worker itself. This
 // function is guaranteed to return a non-nil map.
 func (w *Worker) CanonicalTags(opt ...Option) map[string][]string {
 	dedupedTags := make(map[Tag]struct{})
-	for _, t := range w.apiTags {
+	for _, t := range w.ApiTags {
 		dedupedTags[*t] = struct{}{}
 	}
-	for _, t := range w.configTags {
+	for _, t := range w.ConfigTags {
 		dedupedTags[*t] = struct{}{}
 	}
 	tags := make(map[string][]string)
```

Comment:
It might be nice to move this logic of converting from worker.Tags to a `map[string][]string` onto `worker.Tags`.